### PR TITLE
Add support for file_paths in check_secret.

### PR DESCRIPTION
**Pull request by zados**
Not only services may need to be checked, but also cache files.

### CODE REVIEW

File: src/check_secret.py [hunk 1]
```diff
@@ -91,30 +101,47 @@ def get_time_delta(service_time: datetime,
...
     args = parse_args()
-    delta = get_time_delta(
-        get_service_restart_time(args.service_name),
-        get_secret_file_time(args.timestamp_file_path),
+    if args.service_name:
+        check_delta=get_service_restart_time(args.service_name)
+        warning_name = args.service_name
+    else:
+        check_delta=get_file_mtime(args.file_path)
+        warning_name = 'Server'
```
Comment (TuxPowered42) [inline src/check_secret.py:126]: It's rather if service name … else if file path … else fail "tell me what to check".

File: src/check_secret.py [hunk 2]
```diff
@@ -91,30 +101,47 @@ def get_time_delta(service_time: datetime,
...
+    return datetime.fromtimestamp(path.getmtime(file_path))
 
 
 def main():
     args = parse_args()
-    delta = get_time_delta(
-        get_service_restart_time(args.service_name),
-        get_secret_file_time(args.timestamp_file_path),
+    if args.service_name:
+        check_delta=get_service_restart_time(args.service_name)
```
Comment (kofrezo) [inline src/check_secret.py:122]: A little space for the equals sign here and in the following lines please. It's hard for it to breathe ;-)

**Review by kofrezo — APPROVED** [review]
Early approve as my comment only remarks style issue but please adjust them before merging.

File: src/check_secret.py [hunk 3]
```diff
@@ -91,30 +101,47 @@ def get_time_delta(service_time: datetime,
...
+    """
+    return datetime.fromtimestamp(path.getmtime(file_path))
 
 
 def main():
     args = parse_args()
-    delta = get_time_delta(
-        get_service_restart_time(args.service_name),
-        get_secret_file_time(args.timestamp_file_path),
+    if args.service_name:
```
[the remark this answers is not in the document]
Comment (zados) [reply src/check_secret.py:122]: That is automatically handled by argparse.

**Pull request merged**